glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
Just one update. targetcli was doing a CheckConfig call, but I did the kernel/runner changes to allow addition to be able to return errors, so we can still just drop check_config one day.

Here is the kernel patchset:
https://www.spinics.net/lists/target-devel/msg15657.html

Here is a targetcli fix:
https://github.com/open-iscsi/targetcli-fb/pull/87